### DISCUSSION TITLE
Running into error: getPanelStyle is not a function

### DISCUSSION THREAD

**Discussion by AbhinavChede** [bvaughn/react-resizable-panels#379] [Q&A]
Hi there,

I am relatively new to React and I wanted to use some of the cool features this project offered. After installing this module , I tried running some of the code that is available on the website. I copy pasted the code for [Horizontal Layouts](https://react-resizable-panels.vercel.app/examples/horizontal) and [Vertical Layouts](https://react-resizable-panels.vercel.app/examples/vertical). However, when I run my App.js file, I encounter an error that says "getPanelStyle is not a function". 

After searching a little more, I see that line 161 in this file "react-resizable-panels.browser.development.esm.js" is causing the error. I am using Version 2.0.22 of this library and React v18.3.

I am not sure how to solve this roadblock. Any help would be greatly appreciated.

**Comment by bvaughn** [comment]
Hey @AbhinavChede. Need you to share a repro with me so I can look. This sounds like it's probably something to do with your project configuration (dependency management, bundling, or something) but I can't say.

**Reply by AbhinavChede** [reply]
Hi @bvaughn , thank you for your message.

Here is my dependencies for the project I am trying to implement react-resizable-panels for:
```typescript
"dependencies": {
    "@testing-library/jest-dom": "^5.17.0",
    "@testing-library/react": "^13.4.0",
    "@testing-library/user-event": "^13.5.0",
    "d3": "^7.9.0",
    "d3-drag": "^3.0.0",
    "d3-force-3d": "^3.0.5",
    "fontawesome": "^5.6.3",
    "force-graph": "^1.43.4",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "react-force-graph-2d": "^1.25.2",
    "react-graph": "^0.0.7",
    "react-resizable-panels": "^2.0.22",
    "react-router-dom": "^6.18.0",
    "react-scripts": "5.0.1"
  }
``` 
If you need more information, please let me know. Thank you

**Reply by bvaughn** [reply]
I think I'll need something I can run (CodeSandbox or GitHub project)
